### PR TITLE
Chapter 10: Updating, showing, and deleting users

### PR DESCRIPTION
**Pull request by huynhlv**
Screenshot from 2019-04-04 08-02-11

### CODE REVIEW

**Comment by huynhlv** [comment]
ready

File: app/views/users/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <%= f.email_field :email, class: "form-control" %>
+
+    <%= f.label :password, t(".password") %>
+    <%= f.password_field :password, class: "form-control" %>
+
+    <%= f.label :password_confirmation, t(".confirm") %>
+    <%= f.password_field :password_confirmation, class: "form-control" %>
+
+    <%= f.submit yield(:button_text), class: "btn btn-primary" %>
+<% end %>
```
Comment (VuThiTranVan) [inline app/views/users/_form.html.erb]: thiếu end-line nhé

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= render "form" %>
+    <div class="gravatar_edit">
+      <%= gravatar_for @user %>
+      <a href="http://gravatar.com/emails" target="_blank"><%= t(".change") %></a>
+    </div>
+  </div>
+</div>
```
Comment (VuThiTranVan) [inline app/views/users/edit.html.erb]: end-line chú

**Comment by VuThiTranVan** [comment]
merged
fix comment rồi mợt hì

**Pull request merged**